yaptide/static/openapi.yaml

```diff
@@ -271,6 +319,56 @@ paths:
                 $ref: '#/components/schemas/BasicResponse'
 
 # Jobs Routes
+  /jobs:
+    get:
+      description: Allows for getting job status.
+      parameters:
+        - in: query
+          name: job_id
+          schema:
+            type: string
+      responses:
+        '200':
+          description: TODO
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                  job_state:
+                    type: string
+                    enum: [PENDING, RUNNING, COMPLETED, FAILED]
+                    description: job state
+                  job_task_status:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/JobTaskStatus'
+        '400':
```

Comment:
Can you add some meaningful description of error codes ?
Take a look at spotify API:
![image](https://github.com/yaptide/yaptide/assets/7374733/db208bc0-a95e-4ecd-8d41-8c26bce7614c)

This looks like swagger yaml file
![image](https://github.com/yaptide/yaptide/assets/7374733/b85fc95e-04b9-4cec-a01b-bad7a2411836)

See https://github.com/sonallux/spotify-web-api/blob/main/official-spotify-open-api.yml